initial_bot/mentor_bot_test.py

```diff
@@ -73,6 +74,30 @@ async def on_member_join(member):
             f'Hi {member.name}, welcome to my Discord server!'
         )
 
+    # Save messages that were sent in private mentoring rooms
+    @bot.event
+    async def on_message(message):
+        '''        
+        private mentoring rooms will have some identifier in their names, so the bot can decide whether or not to save the message.
+        For now I am tentatively using "shlomi" as the identifier :-)
+        
+        '''
+        prvt_ment_channels_mark = 'shlomi'
+        channel = message.channel
+        author = message.author
+        if not author.bot and prvt_ment_channels_mark in channel.name:
+            with open(f"initial_bot/{channel.name}_messages.txt", 'a+') as f:
+                date = (message.created_at)
+                f.write(f'In channel id: {channel.id}\n')
+                f.write(f'{date:%d/%m/%Y %H:%M}\n{author.name}\n')
+                f.write(message.content + "\n\n")
+            bot.dispatch('documentation', channel, author.name)
```

Comment:
Eran your code is stunning! ** chef kisses **

Reply:
I want that prvt_ment_channels_mark will be added room creationg function as well, to make sure the room is created with this (maybe on one of the conf yml that @shirganot added)